src/shell/shell_cmds.cpp

```diff
@@ -240,16 +238,36 @@ void DOS_Shell::CMD_HELP(char * args){
 	HELP("HELP");
 	bool optall=ScanCMDBool(args,"ALL");
 	/* Print the help */
-	if (!optall) WriteOut(MSG_Get("SHELL_CMD_HELP"));
+	if (!optall && !*args) WriteOut(MSG_Get("SHELL_CMD_HELP"));
+	upcase(args);
 	Bit32u cmd_index=0,write_count=0;
```

Comment:
Unwrap on separate lines and switch to cstdtype - uint32_t, size_t, int, etc..

Reply:
It now looks like:

``
if (!optall && !*args)
    WriteOut(MSG_Get("SHELL_CMD_HELP"));

I assume this is what you meant. And I have changed "Bit32u" to "uint32_t" as you suggested. Thanks!